config/defaults.js

```diff
@@ -10,5 +10,6 @@ module.exports = {
   'dbPath': './.db',
   'port': 8443,
   'serverUri': 'https://localhost:8443',
-  'webid': true
+  'webid': true,
+  'dataBrowserPath': 'default'
```

Comment:
What are possible alternative values? Why isn't this `static/databrowser.html`?

Reply:
Not quite sure why it was done this way. I do know that it's what activates it.